webhook/admission.go

```diff
@@ -101,15 +101,20 @@ func admissionHandler(rootLogger *zap.SugaredLogger, stats StatsReporter, c Admi
 			patchType = string(*reviewResponse.PatchType)
 		}
 
-		logger.Infof("AdmissionReview for %#v: %s/%s response={ UID: %#v, Allowed: %t, Status: %#v, Patch: %s, PatchType: %s, AuditAnnotations: %#v}",
-			review.Request.Kind, review.Request.Namespace, review.Request.Name,
-			reviewResponse.UID, reviewResponse.Allowed, reviewResponse.Result, string(reviewResponse.Patch), patchType, reviewResponse.AuditAnnotations)
-
 		if !reviewResponse.Allowed || reviewResponse.PatchType != nil || response.Response == nil {
 			response.Response = reviewResponse
 		}
 		response.Response.UID = review.Request.UID
 
+		logger = logger.With(
```

Comment:
`SugaredLogger.With` takes pairs of untyped arguments:

https://godoc.org/go.uber.org/zap#SugaredLogger.With

(Personally, I find the non-sugared logger more intuitive because the types help you out, but we seem to prefer the sugared version.)

Reply:
Thank you for pointing it out. Fixed to use sugared version for both places. Done.